### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by AndrewSirenko**
**Is this a bug fix or adding new feature?**
bugfix

**What is this PR about? / Why do we need it?**
Update OWNERS to include AndrewSirenko as reviewer and approver, as well as remove inactive reviewers. 

**What testing is done?** 
N/A

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -10,9 +10,9 @@ approvers:
 - rdpsin
 - torredil
 - wongma7
+- AndrewSirenko
```
Comment (ConnorJC3) [inline OWNERS:13]: Please move this to the first entry, we like to keep this file in alphabetical order

File: OWNERS [hunk 2]
```diff
@@ -10,9 +10,9 @@ approvers:
...
 reviewers:
 - ConnorJC3
-- gtxu
-- hanyuel
 - rdpsin
 - torredil
+- AndrewSirenko
```
Comment (ConnorJC3) [inline OWNERS:18]: Same as above

File: OWNERS [hunk 3]
```diff
@@ -10,9 +10,9 @@ approvers:
 - rdpsin
 - torredil
 - wongma7
+- AndrewSirenko
 reviewers:
 - ConnorJC3
-- gtxu
-- hanyuel
```
Comment (ConnorJC3) [inline OWNERS:16]: Also drop these from `approvers`

**Review by torredil — APPROVED** [review]
/approve

**Pull request merged**